docs/cli/usage/permission-boundary.md

```diff
@@ -0,0 +1,16 @@
+---
+title: IAM Role Permission Boundary
+description: Apply a Permission Boundary to all IAM Roles created by Amplify CLI.
+---
+
```

Comment:
I think we need more context/introduction, here, re: what this feature does as a general matter / why we added it / and why a user might find it useful. In other words, can we include the user story?